scripts/conversion_tools/unit_conversion.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+"""A pilot version to perform unit conversions. Each conversion must be representable
+as a formula where {var} is substituted by the actual variable (scalar, array) to convert.
+This allows having formulars such as func({var}) where func is defined as an elemental
+function or as an interface to scalar and array-based functions that perform more
+complex conversions than the ones listed here. It is also possible, but in some cases
+less performant, to construct conversions for composed units by combining some of the
+basic conversions listed here. For instance, one could create a speed conversion from
+km h-1 to m s-1 by combining the formulas for km to m and h to min, which will be
+slower than boiling it down to a single mathematical expression (see example below)."""
+
+############
+# Length   #
+############
+
+def mm__to__m():
+    """Convert millimeter to meter"""
+    return '1.0E-3*{var}'
```

Comment:
That's a good point, however I need to either define a default kind in case the variable has an empty kind string (which many variables do at this point), or make the "_" part of the kind='_' + x.kind statement if x.kind is not '').

Reply:
I like the second option.

Reply:
Ok I will do that.

Reply:
Ok, fixed that one.